benchmarks/CMakeLists.txt

```
@@ -122,6 +122,7 @@ add_benchmark(replace src/replace.cpp)
 add_benchmark(search src/search.cpp)
 add_benchmark(std_copy src/std_copy.cpp)
 add_benchmark(swap_ranges src/swap_ranges.cpp)
+add_benchmark(sv_equal src/sv_equal.cpp)

```

These are ordered alphabetically; `sv_equal` should precede `swap_ranges`:
```suggestion
add_benchmark(sv_equal src/sv_equal.cpp)
add_benchmark(swap_ranges src/swap_ranges.cpp)
```